src/cli/commands/lint.ts

```diff
@@ -73,10 +73,6 @@ const lintCommand: CommandModule = {
           alias: 'r',
           description: 'path/URL to a ruleset file',
           type: 'string',
-          coerce(value): Optional<string[]> {
```

Comment:
We allow only one ruleset to be specified at a time.

Reply:
Does this mean `spectral lint --ruleset foo --ruleset=bar` will still work, we just removed `spectral lint --ruleset foo,bar`?